src/components/shared/TextAreaField/index.tsx

```diff
@@ -26,6 +29,7 @@ const TextAreaField = ({
           {label}
         </label>
       )}
+      {helpText && <HelpText>{helpText}</HelpText>}
```

Comment:
We can leave this here for now, but this gets me wondering if we should be even more granular and declarative and break out even thing to it's own component.

In the future it could look like:

```
<Label label="What is your deepest fear" />
<HelpText text="Tell it to me" />
<TextAreaField />
<HelpText  text="2000 characters left" />
```

Reply:
We could also make one form field component to wrap all the others. This component could render the label, help text, etc and take the actual field as a child.

Reply:
I'd like to avoid that for now to avoid prop drilling too deep. The idea I'm going for here is to break down components to the lowest level and use composition rather than adding additional configuration through props.